src/components/SearchTextareaField/SearchTextareaField.test.js

```diff
@@ -51,4 +53,14 @@ describe('Given SearchTextareaField', () => {
       expect(onChange).toHaveBeenCalled();
     });
   });
+
+  describe('when typing textarea', () => {
```

Comment:
By the test names you could read how application works, it's kind a documentation. You can combine it with previous tests

```
// this test describes to user that what should happened when user typing inside textarea
 describe('when typing inside textarea', () => {
    // here user see what should happend after typing
    it('should handle onChange', () => {
      const { getByTestId } = renderSearchTextareaField();
      fireEvent.change(getByTestId('search-textarea'), { target: { value: 'test' } });
      expect(onChange).toHaveBeenCalled();
    });
    
    //after that you can continue the logic
    describe('when user press "Enter" button', () => {
     it('should handle form submit', () => { // or should handle onSubmitSearch action
     ...
```